src/docs/asciidoc/jcache/icache.adoc

```diff
@@ -27,17 +27,8 @@ read from and write to ICache.
 
 A `CacheManager`, started either as a client or as an embedded member,
 can be configured to start a new Hazelcast instance or reuse an already
-existing one to connect to a Hazelcast cluster. To achieve this, request
-a `CacheManager` by passing a `java.net.URI` instance to `CachingProvider.getCacheManager()`.
-The `java.net.URI` instance must point to either a Hazelcast configuration or to the name of a named
```

Comment:
`URI` as Hazelcast config file still works, we should keep this part. So `getCacheManaget(new URI("file:///home/vb/hazelcast.xml"), ...)` still works. Passing an existing instance name as `URI` is no longer supported and should be removed.